### DISCUSSION TITLE
Data challenge

### DISCUSSION THREAD

**Discussion by EstherPlomp** [EstherPlomp/TNW-RDM-101#7] [General]
What is your main challenge regarding your data management? Please respond below with your data challenge!

Please read your peers’ responses and reply to any you find interesting.

**Comment by RamonvanValderen** [comment]
One of the main challenges in data management for my project is to obtain and maintain a proper overview of all the simulations I've created. Creating a workflow where I can clearly document why I created a certain simulation and which parameters/methods I used will be key for proper RDM, especially since there will also be much trial-and-error to make the simulations work. Furthermore, documenting clearly the post-processing of the simulation results will be very important as the post-processing combines open source commercial software combined with programming files for further data manipulation and visualization. Creating a documentation method to clearly document these data processing pipelines is still something I have to figure out.

**Reply by EstherPlomp** [reply]
I hope your folder structure will help you partly with organising the data. From the sign ups from this course we do not have other people working with Computational Fluid Dynamic (CFD) data (feel free to respond and correct me people!) - but I know that at ChemE there's the [Transport Phenomena group ](https://www.tudelft.nl/tnw/over-faculteit/afdelingen/chemical-engineering/about-the-department/transport-phenomena) that is working on this. I can try to reach out to them to see if perhaps one of their PhD candidates is interested in exchanging experiences with you? (After your holidays of course).

**Comment by rwmulder** [comment]
I am working with a four people on one setup. A main challenge for us is to keep track of eachothers work and progress without wasting a lot of time in meetings. By keeping informative and reproducible logs we could become much more efficient. Furthermore it will help later on in the writing process. In this course I am hoping to hear some new ideas of how to improve our workflow.

**Reply by EstherPlomp** [reply]
That sound challenging indeed!

Have you tried setting up collaborative notes (for example Google documents or if you're working with sensitive data work on a document in Teams) or setting up documentation on GitLab/GitHub? You could use the issues to keep track of shared concerns and perhaps bookings? GitHub also has 'Kanban' type project boards where you can keep track of tasks. Alternative tools might be Asana (I don't think we have TU Delft licenses for this but can check if it sounds interesting). 

Anyone else that has any tips for Roland?

**Comment by bertvanherck** [comment]
I started 6 months ago. I do not have a lot of data yet. But now I want to establish a clear and efficient way of structuring my data that I can continue using during my entire Ph.D. without having issues with finding, understanding, ... data. I think that labeling all data and providing short tags/labels will be essential but challenging to still understand my data several months after I did acquire it.

**Reply by EstherPlomp** [reply]
It can be difficult to think about how to organise the data when you don't have a lot yet. On the bright side: it will also be easier to organise things from the start because you don't have to clean up :). Hopefully the [Realising FAIR: Organisation](https://estherplomp.github.io/TNW-RDM-101/08-Realising-FAIR-Organisation.html) part of the course provides you with some tools/tips to organise your data efficiently!

**Comment by Nickterveer** [comment]
I am mainly working on experimental techniques during my PhD, but as a side project when my experiments are running or we are waiting for chemicals/parts, I work on MD simulations. This generates huge amounts of data (50 GB per day), which I have to structure smartly. So that even after a few week break, I can continue where I left off.

**Reply by EstherPlomp** [reply]
Does your storage location provide you with sufficient space if you generate 50 GB a day? (Project Drive is extendable, so probably the best long-term location for this data).

**Comment by MBeatrizDiniz** [comment]
One of my current main challenges related to data management is to fully understand and get access to previous work related to my project.

**Reply by EstherPlomp** [reply]
Do you already have an overview of the researchers that you need to contact for this? Or the work that you need to find/read/analyse? It can be helpful to establish an overview and set certain milestones so that it is not too overwhelming.

**Comment by annickteepe** [comment]
My raw measurement is properly saved and backed up. However, I struggle with how to save analysed data, analysis scripts, and the data of 'try out stuff' in the lab in a structured and organised way

**Reply by EstherPlomp** [reply]
Hopefully the [Realising FAIR: Organisation](https://estherplomp.github.io/TNW-RDM-101/08-Realising-FAIR-Organisation.html) part of the course provides you with some ideas/suggestions for this. It also doesn't hurt to have a 'try out' folder that is a bit more messy - you'll need to start somewhere to figure out how it can be easier organised.

**Comment by maverheijen** [comment]
Storing raw experimental data will be relatively simple, although I have to use consistent file naming to keep a clear overview. My biggest challenge with RDM during my PhD will most likely relate to keeping a good structure in the way I store code, analysed data and any other scripts or files I use to design experiments. Also, ensuring a good structure to keep track of such files and raw data of the students I supervise.

**Reply by EstherPlomp** [reply]
Hopefully the [Realising FAIR: Organisation](https://estherplomp.github.io/TNW-RDM-101/08-Realising-FAIR-Organisation.html) part of the course provides you with some suggestions for folder structures/organisation. There is no golden standard of course - as long as you're consistent it works. Perhaps you can also ask feedback of the students to see if they can find/understand the structure - adjustments may be needed if they have trouble storing things in the right places.

**Comment by sweerdenburg1** [comment]
I synthesize and characterize many materials. The procedures for synthesis measurements will affect the conclusions that are drawn for all my materials. It is therefore important to keep a good record of all steps that were followed and to link that to the collected datasets. My challenge is therefore to store my data in a logical way, so that it is easy to see what procedures were followed to obtain that data.

**Reply by EstherPlomp** [reply]
Hopefully the [Realising FAIR: Data organisation](https://estherplomp.github.io/TNW-RDM-101/08-Realising-FAIR-Organisation.html) and [Realising FAIR: Data Documentation](https://estherplomp.github.io/TNW-RDM-101/09-Realising-FAIR-Documentation.html) will provide you with some pointers!

**Comment by SDev5** [comment]
During a typical project I will generate and work (among others) with simulation code and data, measurement data and analysis code. Now, it often happens that the different types of data are stored in different places. The main challenge for me is to construct an organized system with a clear file structure and clear file names, which makes it easy to find all the data related to a specific project (from simulations to design to measurement and analysis). Another challenge for me is properly storing simulation data in an organized way with proper documentation.

**Reply by EstherPlomp** [reply]
Next to the pointers in [Realising FAIR: Data organisation](https://estherplomp.github.io/TNW-RDM-101/08-Realising-FAIR-Organisation.html) and [Realising FAIR: Data Documentation](https://estherplomp.github.io/TNW-RDM-101/09-Realising-FAIR-Documentation.html), it might be needed to discuss this with the rest of your team. If you have to work collaboratively it can be helpful if the group follows the same convention/organisation/documentation styles instead of everyone reinventing the wheel.

**Comment by begum94** [comment]
Proper documentation for all of your data is quite challenging in my opinion. By proper documentation, I mean experimental details, raw data, processing scripts, tables and figures and maybe even some conclusions and discussions on the data. It seems easy at the beginning, but as time passes, being consistent and being up to date becomes harder. I want to do this without spending unnecessary time on this as well. Data storage and management in long term, for example having access to 5 year old raw data and being able to obtain reasonable results with experimental details out of stored info is not an easy task, but I would like to achieve that.

**Reply by EstherPlomp** [reply]
Agree that data documentation is a significant challenge! Hopefully you'll find the information in [Realising FAIR: Data Documentation](https://estherplomp.github.io/TNW-RDM-101/09-Realising-FAIR-Documentation.html) helpful!

**Comment by Azinkarimzad** [comment]
As I work with heavy datasets, models, and various coding languages, effective data management is crucial for my project. Additionally, my project is part of the Delphi consortium, and I must provide clients with regular reports twice a year. Therefore, data management planning is an essential part of my daily tasks.

To manage my data more efficiently, I am trying to implement a few strategies. For example, I store data securely in cloud storage solutions like GitLab and surf drive and an external HDD. I do also plan to use consistent naming  and maintain detailed documentation of my data.

**Reply by EstherPlomp** [reply]
Sounds you're already well on the way! I hope the Realising FAIR pages will provide you with helpful information in the upcoming week!

**Comment by TamaraJankovic03** [comment]
The main challenge regarding data management for me is to properly document all the simulations that I have made. Sometimes creating a simulation can be trial-and-error work, which means that it is very important to have all the trials well documented. Additionally, I will analyze all simulations, especially the successful ones and I need to document the results of those analyses in a structured way. If the results are well organized, it will be easier to choose the simulation with the optimal result.

**Reply by EstherPlomp** [reply]
Sounds like some work! In the [Realising FAIR: Data Documentation](https://estherplomp.github.io/TNW-RDM-101/09-Realising-FAIR-Documentation.html) there's a link to some other materials that are not part of the course but might be of help for this: [Code Refinery materials on Code Documentation](https://coderefinery.github.io/documentation/).

**Comment by bejuarez96** [comment]
I consider that the main challenge for me would be to save and backup my data from my physical lab journal to a digital lab journal. Since most of my data will come from the set-up and results of my experiments, I need to look for the proper way to transfer my raw data to a digital form to be able to analyze it in the future without any problem. Furthermore, my data should also be clear enough to draw conclusions and be able to continue with the research.

**Reply by EstherPlomp** [reply]
You might want to have a look at the TU Delft electornic lab notebook licenses for this. See the information in [Realising FAIR: Data Documentation](https://estherplomp.github.io/TNW-RDM-101/09-Realising-FAIR-Documentation.html), 10.4. It is also fine to keep a word/text file of the digital version - as long as you have some digital version available that you can back up you should be good to go! A search function is very helpful in my personal experience when you try to look for specific samples/dates.

**Comment by MarijnVDBr** [comment]
My main challenge is to link all experimental protocols, analog lab notes, raw data files, processed data files and results (graphs, overviews) to each other. The challenge is to provide a clear overview of the main results obtained in a certain period, while the raw data files can still be found and how they are processed.
It would also be good to already think about what data is required to be available for publication (what should be attached to the publication, and what should be available on request).

**Reply by EstherPlomp** [reply]
Indeed a challenge! Have you tried out an [electronic lab notebook](https://estherplomp.github.io/TNW-RDM-101/09-Realising-FAIR-Documentation.html#assignment-electronic-lab-notebooks-optional) for this? 

Otherwise a README file is helpful to record this information, or perhaps an excel sheet. 

In terms of what is data is required to be publicly shared: from our Faculty policy on Research Data Management: 
_"The minimal requirement is the deposition of the numerical data (processed data) underlying the figures in academic papers and PhD theses"_ 
More is encouraged, but at this moment not required :). See also [6.8 RDM responsibilities](https://estherplomp.github.io/TNW-RDM-101/05-Essentials-for-Research-Data.html#rdm-responsibilities).

**Comment by phvree** [comment]
My main challenges are in saving not only the raw data files, but also my measurement scripts, measurement parameters not directly in my code and scripts I use to analyze the data, in a way that is efficient and also flexible enough, so that I can reproduce and understand my work at a later time

**Reply by EstherPlomp** [reply]
Hopefully you find the information in [Realising FAIR: Data Documentation](https://estherplomp.github.io/TNW-RDM-101/09-Realising-FAIR-Documentation.html) helpful for this!

**Comment by cfchen98** [comment]
My main challenges are how to organize, store and backup the data (raw data and processed data) in a reasonable way. In addition, most of the data I get is experimental data, and the reproducibility is quite important, which means I should not only be able to find the data I need, but also make very detailed labnote to make it repeatable.

**Reply by EstherPlomp** [reply]
I hope the information in [Realising FAIR: Data organisation](https://estherplomp.github.io/TNW-RDM-101/08-Realising-FAIR-Organisation.html) and [Realising FAIR: Data Documentation](https://estherplomp.github.io/TNW-RDM-101/09-Realising-FAIR-Documentation.html) will help you with this! Its probably helpful to check the information on [electronic lab notes](https://estherplomp.github.io/TNW-RDM-101/09-Realising-FAIR-Documentation.html#assignment-electronic-lab-notebooks-optional) particularly.

**Comment by arr0w-hs** [comment]
I will be working mostly with codes and analytical data. So, the main challenge that I have would be proper organisation of all the data so that it is ordered well, and I can look back at it without much trouble when needed in the future.

**Reply by EstherPlomp** [reply]
I hope the information in [Realising FAIR: Data organisation](https://estherplomp.github.io/TNW-RDM-101/08-Realising-FAIR-Organisation.html) and [Realising FAIR: Data Documentation](https://estherplomp.github.io/TNW-RDM-101/09-Realising-FAIR-Documentation.html) can help you with this in the upcoming week!

**Comment by MiriamCammaert** [comment]
My main challenge regarding my data management is to keep a well-documented overview of why and how data was obtained. This will not only be extremely important for myself to keep track of my project, but also for other PhD candidates that will need to access my data since I am working as part of a consortium.

**Reply by EstherPlomp** [reply]
The pointers in [Realising FAIR: Data Documentation](https://estherplomp.github.io/TNW-RDM-101/09-Realising-FAIR-Documentation.html) will hopefully help you with this. As mentioned to @rooswouters  as well (see below) - the only way to know for sure if others can find their way around your data is to do a test run and see where they succeed or fail! :)

**Comment by rooswouters** [comment]
My main challenge in data management is currently storing and processing my data in a manner that I (and other people) will be able to understand my results and thinking process, even after several months/years.

**Reply by EstherPlomp** [reply]
The information in [Realising FAIR: Data organisation](https://estherplomp.github.io/TNW-RDM-101/08-Realising-FAIR-Organisation.html) and [Realising FAIR: Data Documentation](https://estherplomp.github.io/TNW-RDM-101/09-Realising-FAIR-Documentation.html) should help you with this! Next to that, to ensure that other people will also find it understandable - it is good to do a test run with a colleague or your supervisor to see if they are able to find their way through your structures/names!

**Comment by MaximDeSmet** [comment]
My main data management challenge is to maintain a proper overview and documentation on all collected data. Organizing data with clear metadata will ensure I will be able to find my old data easily and know exactly how it was gathered.

**Reply by EstherPlomp** [reply]
I hope you find the information in the [Realising FAIR: Data organisation](https://estherplomp.github.io/TNW-RDM-101/08-Realising-FAIR-Organisation.html) and [Realising FAIR: Data Documentation](https://estherplomp.github.io/TNW-RDM-101/09-Realising-FAIR-Documentation.html) sections helpful for this!

**Comment by yzhang0429** [comment]
My data challenge is to build up a proper storage and organization structure for different type of data. In my research there are different kinds of data like design files, pictures, measurement code and database. They are stored in different folders and sometime it's hard to look up. I would like to figure out a proper way to arrange the storage structure to make it more efficiency to look up later.

**Reply by EstherPlomp** [reply]
I hope you find the information in the [Realising FAIR: Data organisation](https://estherplomp.github.io/TNW-RDM-101/08-Realising-FAIR-Organisation.html) useful to organise everything!
